.github/workflows/prettier.yml

```diff
@@ -0,0 +1,20 @@
+name: Prettier
+on: push
+jobs:
+  run-linters:
+    name: Run Prettier
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out Git repository
+      uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+        node-version: 12
+      - name: Change to UI folder
+        run: cd ui
+      # ESLint and Prettier must be in `package.json`
+      - name: Install Node.js dependencies
+        run: yarn
+      - name: Run linters
```

Comment:
Is it a waste to make a whole flow for this? Running prettier is pretty quick isn't it? Can't we just add it to `yarn build` in the ui?

Reply:
Also this is only running it on .js files

Reply:
Yea. We should just run prettier before `yarn build`